Snakefile

```diff
@@ -18,3 +18,4 @@ root -l -b -q -e '.L {input}+'
 """
 
 include: "benchmarks/diffractive_vm/Snakefile"
+include: "benchmarks/u_rho/Snakefile"
```

Comment:
Please, include the tex file for benchmarks/u_rho/BenchmarkPlotsExplained.pdf, maybe we can generate PDF on CI and store it artifacts along with the plots.

Reply:
Okay I've now updated the latex file! I've tried the `pdflatex` command in eicshell to compile it and that didn't work. How can we generate the PDF on CI?

Reply:
I've pushed a hacky solution. Perhaps, we need to request some small latex engine to be added to the container.